azure-communication-ui/azure-communication-ui/src/main/res/layout/azure_communication_ui_bottom_drawer_title_cell.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+   Copyright (c) Microsoft Corporation. All rights reserved.
+   Licensed under the MIT License.
+  -->
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:focusable="false"
+    >
+    <TextView
+        android:id="@+id/cell_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="18dp"
+        android:layout_marginEnd="18dp"
+        android:ellipsize="end"
+        android:singleLine="true"
+        android:text="@string/azure_communication_ui_audio_device_drawer_android"
+        android:textColor="@color/azure_communication_ui_color_text_primary"
```

Comment:
Is this a color we have already? is it just white/black based on theme?

It should be a theme attribute, probably a built in material one like "?attr/colorOnSurface"

If it's not new, just leave it for now. We can migrate hard coded colors to theme attributes later.

Reply:
This isn't new, I just moved the color from the existing kotlin code to the XML layout, since this was no longer necessary to be set dynamically.